TCAT/Cells/ServiceAlertTableViewCell.swift

```diff
@@ -0,0 +1,238 @@
+//
+//  ServiceAlertTableViewCell.swift
+//  TCAT
+//
+//  Created by Omar Rasheed on 12/7/18.
+//  Copyright © 2018 cuappdev. All rights reserved.
+//
+
+import UIKit
+
+class ServiceAlertTableViewCell: UITableViewCell {
+
+    static let identifier: String = "serviceAlertCell"
+    private let fileName: String = "serviceAlertTableViewCell"
+    var alert: Alert?
+    var rowNum: Int!
+    
+    let borderInset = 16
+    
+    var timeSpanLabel: UILabel!
+    var descriptionLabel: UILabel!
+    var affectedRoutesLabel: UILabel!
+    var affectedRoutesStackView: UIStackView?
+    var topSeparator: UIView?
+    
+    override init(style: UITableViewCell.CellStyle, reuseIdentifier: String?) {
+        super.init(style: style, reuseIdentifier: reuseIdentifier)
+        
+        setupTimeSpanLabel()
+        setupDescriptionLabel()
+    }
+    
+    func setData() {
+        if let fromDate = alert?.fromDate, let toDate = alert?.toDate {
+            timeSpanLabel.text = formatTimeString(fromDate, toDate: toDate)
+        }
+        descriptionLabel.text = alert?.message
+        if let routes = alert?.routes, !routes.isEmpty {
+            setupAffectedRoutesStackView()
+            setupaffectedRoutesLabel()
+        }
+        
+        if rowNum > 0 {
+            setupTopSeparator()
+        }
+    }
+    
+    private func setupTimeSpanLabel() {
+        
+        timeSpanLabel = UILabel()
+        timeSpanLabel.numberOfLines = 0
+        timeSpanLabel.font = .getFont(.semibold, size: 18)
+        timeSpanLabel.textColor = Colors.primaryText
+        
+        contentView.addSubview(timeSpanLabel)
+    }
+    
+    private func setupDescriptionLabel() {
+        
+        descriptionLabel = UILabel()
+        descriptionLabel.numberOfLines = 0
+        descriptionLabel.font = .getFont(.regular, size: 14)
+        descriptionLabel.textColor = Colors.primaryText
+        
+        
+        contentView.addSubview(descriptionLabel)
+    }
+    
+    private func setupaffectedRoutesLabel() {
+        
+        affectedRoutesLabel = UILabel()
+        affectedRoutesLabel.font = .getFont(.semibold, size: 18)
+        affectedRoutesLabel.textColor = Colors.primaryText
+        affectedRoutesLabel.text = "Affected Routes"
```

Comment:
We've started using `Constants.swift` for all client-facing strings. Add this there!